### DISCUSSION TITLE
Dinâmicas de grupo na empresa

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#519] [Discussão]
Fala galera beleza?! Vocês fazem alguma dinâmica de grupo na empresa em que trabalham?!
Algo pra fortalecer os laços da equipe ou ajustar pontos que não estão muito legais?!

**Comment by podrivo** [comment]
A famigerada cervejinha. O melhor lubrificante social que existe, hehe. Tem também comemorações de aniversário, Hackathons esporádicos, festas de trimestre e final de ano. Pra colher feedback e ajustar pontos, existem reuniões individuais mensais com o gestor da área e o RH está sempre presente pra escutar o que for.
